src/index.ts

```diff
@@ -341,6 +333,46 @@ export default <ExportedHandler<Env>>{
 						});
 					}
 					const message = await res.json<APIMessage>();
+					// Create a thread from this message, asking the user for more information
+					const thread = await fetch(`${RouteBases.api}${Routes.threads(message.channel_id, message.id)}`, {
+						headers: {
+							Authorization: `Bot ${env.DISCORD_BOT_TOKEN}`,
+							"Content-Type": "application/json",
+						},
+						method: "POST",
+						body: JSON.stringify({
+							name: threadName ?? question.substring(0, 17) + "...",
+						}),
+					});
+					const promises = [];
+					if (thread.ok) {
+						promises.push(fetch(`${RouteBases.api}${Routes.channelMessages(message.id)}`, {
+							headers: {
+								Authorization: `Bot ${env.DISCORD_BOT_TOKEN}`,
+								"Content-Type": "application/json",
+							},
+							method: "POST",
+							body: JSON.stringify({
+								content: `Question posted by: <@${interaction.member?.user.id}>`,
+							}),
+						}));
```

Comment:
I know I originally said to send a message - but it may be a little less cluttered to simply add them to the thread via the API? <https://discord.com/developers/docs/resources/channel#add-thread-member>